src/browser/api/webcontents.ts

```diff
@@ -0,0 +1,86 @@
+import * as url from 'url';
+
+export function executeJavascript(webContents: Electron.WebContents, code: string, callback: (e: any, result: any) => void): void {
+    webContents.executeJavaScript(code, true, (result) => {
+        callback(undefined, result);
+    });
+}
+
+export function getInfo(webContents: Electron.WebContents) {
+    return {
+        canNavigateBack: webContents.canGoBack(),
+        canNavigateForward: webContents.canGoForward(),
+        title: webContents.getTitle(),
+        url: webContents.getURL()
+    };
+}
+
+export function getAbsolutePath(webContents: Electron.WebContents, path: string) {
+    const windowURL = webContents.getURL();
+    return  url.resolve(windowURL, path);
+}
+
+export function navigate (webContents: Electron.WebContents, url: string) {
```

Comment:
minor detail, but spacing consistency after function names. Feel free to ignore.